.prettierrc

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Single ticks are the devil. Shall we use https://www.npmjs.com/package/@projectsophon/prettier-config

Reply:
hm im trying to use this in my package.json and it just doesnt work at all, not sure why, maybe my prettier/vscode is old version but it doesnt seem like it

Reply:
im just going to leave it in repo for now since that seems easier fix